pyaedt/filtersolutions_core/lumped_termination_impedance.py

```diff
@@ -0,0 +1,391 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from ctypes import POINTER
+from ctypes import byref
+from ctypes import c_bool
+from ctypes import c_char_p
+from ctypes import c_int
+from ctypes import create_string_buffer
+from enum import Enum
+
+import pyaedt
+
+
+class ComplexTerminationDefinition(Enum):
+    """Selects type of complex presentation.
+    Attributes:
+    POLAR: Represents polar definition.
+    CARTESIAN: Represents Cartesian definition.
+    PARALLEL: Represents parallel definition with real entry parallel to imaginary entry.
+    REAL: Represents only real impedance definition.
+    """
+
+    POLAR = 0
+    CARTESIAN = 1
+    PARALLEL = 2
+    REAL = 3
+
+
+class ComplexReactanceType(Enum):
+    """Selects type of complex impedance as reactance, equivalent inductance, or equivalent capacitance.
+
+    Attributes:
+    REAC: Represents pure reactance of complex impedance.
+    IND: Represents equivalent inductance in henries.
+    CAP: Represents equivalent capacitance in farads.
+    """
+
+    REAC = 0
+    IND = 1
+    CAP = 2
+
+
+class TerminationType(Enum):
+    """Selects either source or load complex impedance table.
+
+    Attributes:
+    SOURCE: Represents source impedenace table.
+    LOAD: Represents load impedenace table.
+    """
+
+    SOURCE = 0
+    LOAD = 1
+
+
+class LumpedTerminationImpedance:
+    """Manipulates access to the entries of source and load complex impedance tables.
+
+    This class allows you to enter, edit or remove the entries of source and load complex impedance tables.
```

Comment:
```suggestion
    This class allows you to enter, edit, or remove the entries of source and load complex impedance tables.
```
Use Oxford comma. In a list of three or more items, place a comma before the "and" or the "or" in the list of items.